trade-finance-manager-api/tsconfig.eslint.json

```diff
@@ -1,11 +1,8 @@
 {
     "extends": "./tsconfig.json",
     "include": [
-        "server",
+        "src",
         "api-tests",
-        "component-tests",
-        "scripts",
-        "test-helpers",
```

Comment:
Good catch, this directories don't exist so I'm guessing this tsconfig was originally copied across from another service and not changed or something.

Reply:
Yep I think exactly that